code/game/objects/items/uav.dm

```diff
@@ -0,0 +1,347 @@
+#define UAV_OFF 0
+#define UAV_ON 1
+#define UAV_PAIRING 2
+#define UAV_PACKED 3
+
+/obj/item/device/uav
+	name = "recon skimmer"
+	desc = "A semi-portable reconisance drone that folds into a backpack-sized carrying case."
```

Comment:
Ack, I didn't catch this when looking over the original PR - 'reconnaissance' is the correct spelling

Reply:
mlem